src/modules/dashboard/components/Dialogs/WrongNetworkDialog/WrongNetworkDialog.tsx

```diff
@@ -25,8 +27,17 @@ const MSG = defineMessages({
 const displayName = 'dashboard.ColonyHome.WrongNetworkDialog';
 
 const WrongNetworkDialog = ({ cancel }: DialogProps) => {
+  const { networkId, ethereal } = useLoggedInUser();
+  const isNetworkAllowed = checkIfNetworkIsAllowed(networkId);
+
   const networkName = NETWORK_DATA[process.env.NETWORK || DEFAULT_NETWORK].name;
```

Comment:
If we're now supporting Ethereum and Goerli as well, would it make sense to update this message from `Please connect to {networkName}` to `Please connect to a supported network` (and perhaps a link to a list of supported networks?

Reply:
We support Ethereum and Goerli in the sense that theoretically the Dapp could be deployed in one of those networks and still work, however, currently the Dapp will only allow you to do stuff if your wallet is connected to the network that the Dapp is deployed in (Gnosis Chain in our case).

This is reflected in the following conditional inside `checkIfNetworkIsAllowed`:
```
// checks if user has a supported wallet
walletSupportedNetwork && 
// checks if it's the same network as the Dapp's
(walletSupportedNetwork.chainId === currentNetworkData.chainId ||
// or if it's on local dev then disregard that
  onLocalDevEnvironment)
```